 main.tf

```diff
@@ -70,6 +70,8 @@ resource "azurerm_app_service" "main" {
         "FLASK_APP"="todo_app/app"
         "FLASK_ENV"="development"
         "FLASK_SKIP_DOTENV"="True"
+        "LOGGLY_TOKEN"= var.LOGGLY_TOKEN
+        "LOG_LEVEL"="DEBUG"
```

Comment:
Worth considering whether you want to set the production app to DEBUG as a default - maybe INFO would be better suited? Very minor, but it also could be worthwhile changing this to use a variable with a default value set - you'll still get the advantage of not needing to specify it, but you'll get the flexibility if you want to.